dockstore-client/src/main/java/io/dockstore/common/FileProvisioning.java

```diff
@@ -239,6 +239,11 @@ public void provisionInputFiles(String parameterFilePath, List<Pair<String, Path
         executorService.shutdownNow();
     }
 
+    public static URI createInputFileIdentifier(String filepath) {
```

Comment:
You can make this protected, because the test class is in the same package.

Reply:
I would also suggest renaming this to more accurately indicate its function, so you don't have to read the function to understand what it does. Maybe something like `createURIFromUnencodedPath`, or half-jokingly, but maybe not, `createURIFromPathThatMightHaveSpaces`.

I personally, although I don't think it's a team standard and I know I can be overly verbose, would also reference the GitHub issue in a comment so the casual peruser would know exactly why this is here. Or if not here, where you invoke the function.